initfiles/sbin/backup.sh.in

```diff
@@ -55,7 +63,9 @@ fi
 
 log "${header} Backup will occur every ${interval} hours"
 while true; do
-    sleep ${interval}m
+    # background for non blocking wait
+    sleep ${interval}h &
+    wait
```

Comment:
what happens if interval is long and a request to stop the backupnode service comes in here whilst it's waiting?